frontend/src/pages/project/Settings/Stages.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <FormHeading>Pipeline Stages</FormHeading>
+    <FormRow>
+        <template #description>
+            <p class="mb-3">Here, you can configure a "Stage". This will enable you to deploy the updates to this project, directly onto the next "Stage" in your development Pipeline.</p>
+            <p class="">This feature is often used to create a Dev > Staging > Production pipeline, where each Stage of your pipeline is a standalone Project.</p>
+        </template>
+        <template #input>&nbsp;</template>
+    </FormRow>
+
+    <FormRow v-model="input.target" :options="projects" data-el="target-project">
+        <template v-slot:default>Target Project</template>
+    </FormRow>
+
+    <div class="mt-6 flex gap-4">
+        <ff-button :disabled="!input.target || loading" @click="deploy()" data-action="push-stage">
+            {{ deploying ? 'Pushing Stage...' : 'Push to Stage' }}
+        </ff-button>
+        <ff-button kind="secondary" :to="{name: 'Project', params: { 'id': input.target }}" :disabled="!input.target" data-action="view-target-project">
+            View Target Project
+        </ff-button>
+    </div>
+</template>
+
+<script>
+import Alerts from '@/services/alerts'
+import Dialog from '@/services/dialog'
+
+import ProjectAPI from '@/api/project'
+import TeamAPI from '@/api/team'
+
+import FormHeading from '@/components/FormHeading'
+import FormRow from '@/components/FormRow'
+
+export default {
+    name: 'ProjectSettingsStages',
+    props: ['project'],
+    data: function () {
+        return {
+            projects: [],
+            deploying: false,
+            input: {
+                target: null
+            }
+        }
+    },
+    watch: {
+        project: function () {
+            this.loadProjects()
+        }
+    },
+    mounted () {
+        this.loadProjects()
+    },
+    methods: {
+        async deploy () {
+            const msg = {
+                header: 'Push to Stage',
```

Comment:
"Deploying to ${parts.target}"

Reply:
The next sentence even talks about a target, Deploying to target project, that works too